src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,81 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 1 << 4;
+    static final float LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] table;
+    private int size;
+    private float threshold;
```

Comment:
```suggestion
    private int threshold;
```
There may be incorrect comparisons with size.

Reply:
checked, but if DEFAULT_CAPACITY =16  and grow, threshold will be integer